MeetU/MeetU/Scripts/Controllers/Meetup/edit.js

```diff
@@ -15,6 +15,7 @@
                 vm.editParams = meetup
             });
         vm.submitForm = function () {
+            vm.editParams.where = document.querySelector('.js-meetup-edit-where').value;
```

Comment:
generally looks ok, but does that work after you remove this line? if it doesn't work, why did you do this refactor?